### ISSUE TITLE
Logging disabled when running 2 MantidPlots in parallel

### ISSUE THREAD

**Issue by josephframsay** [mantidproject/mantid#21159]
On Windows 7, opening an instance of MantidPlot while a second is already open causes an error in the logging framework in the new instance (`Error in logging framework: Cannot open file`). This is because the two versions are fighting for the same log file in AppData/mantidproject/mantid

### Steps to reproduce the behavior
Open two instances of MantidPlot, and look for the above error in the second.

### Platforms affected
Windows 7

**Comment by DavidFair** [comment]
This is not an issue on Workbench

Since this is a Windows / MantidPlot issue which has always existed and were mothballing MantidPlot I'm going to close this as fixed / wontfix respectively
